minho/p,y개수.js

```diff
@@ -0,0 +1,10 @@
+function solution(s){
+    var answer = true;
+    var count_p=0;
+    var count_y=0;
+    for(var i=0;i<s.length;i++){
+        if(s[i]=='p'||s[i]=='P')count_p++;
+        if(s[i]=='y'||s[i]=='Y')count_y++;
+    }
+    return (count_p==count_y)?true:false;
```

Comment:
```suggestion
    return count_p === count_y ? true : false;
```
== 보단 타입과 값까지 비교해주는 ===을 쓰는 것이 좋아요!